packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -179,7 +179,8 @@ export enum FieldMetadataType {
   Relation = 'RELATION',
   Select = 'SELECT',
   Text = 'TEXT',
-  Uuid = 'UUID'
+  Uuid = 'UUID',
+  PipelineSteps = 'PIPELINE_STEPS'
```

Comment:
PIPELINE_STEP should be singular as a fieldType. Could you replace pipelineSteps by pipelineStep everywhere in your code when talking about the field type?